charts/prompt-proto-service/values.yaml

```diff
@@ -71,6 +71,15 @@ apdb:
   # @default -- None, must be set
   namespace: ""
 
+alerts:
+  # -- Username for sending alerts to the alert stream
+  username: "kafka-admin"
```

Comment:
Do you really want to set a default for `username`?  Maybe better leaving it unset and setting it in each instrument's service? 

Same for `server`.

Reply:
The server will be the same for all services, and at the moment we only have one admin account. This may change in the future but for now the setup is that everything but the topic is the same for each service. We can do it that way, but it isn't needed right now.